pandas/tests/frame/methods/test_replace.py

```diff
@@ -1403,3 +1403,28 @@ def test_replace_with_duplicate_columns(self, replacement):
         result["B"] = result["B"].replace(7, replacement)
 
         tm.assert_frame_equal(result, expected)
+
+    def test_replace_period_ignore_float(self):
+        """
+        Regression test for GH#34871: if df.replace(1.0, 0.0) is called on a df
+        with a Period column the old, faulty behavior is to raise TypeError
+        (reported for Pandas 1.0.4). The intended behavior is to just ignore
+        that column. This has been fixed in newer versions. This is a regression
+        test that triggers for the old, broken behavior, i.e.
+        TypeError is raised
+        """
+        df = pd.DataFrame({"Per": [pd.Period("2020-01")] * 3})
```

Comment:
please use explicit comparisons, if this is *only* a test which is expected to fail then use ```@pytest.xfail(reason=....)```

Reply:
IOW don't comment out code

Reply:
Hey! It's sort of the opposite, I expect being able to call df.replace(...) without it raising an exception for these specific inputs.

Would I then simply run "df.replace(...)" in the unit test, and that's sufficient?

Reply:
you would need to assert the results

Reply:
updated, code now asserts expected result, also removed commented-out blocks of code